__tests__/app.test.js

```diff
@@ -25,3 +26,59 @@ describe.only('GET /api/topics', () => {
 			})
 	})
 })
+describe('GET /api/articles', () => {
+	test('Should return a list of articles', () => {
+		return request(app).get('/api/articles')
+			.expect(200)
+			.then( (response) => {
+				expect(response.body.articles.length).toBe(12)	
+				response.body.articles.forEach( (article) => {
+					expect(article).toEqual(expect.objectContaining({
+						author: expect.any(String),
+						title: expect.any(String),
+						article_id: expect.any(Number),
+						topic: expect.any(String),
+						created_at: expect.any(String),
+						votes: expect.any(Number), 
+						comment_count: expect.any(String),
+					}))
+			})
+			})
+	})
+})
+
+describe('GET /api/articles/:article_id', () => {
+	it('Should return a specific article in response to a GET request at the parametric endpoint', () => {
+		return request(app)
+			.get('/api/articles/1')
+			.expect(200)
+			.then( (response) => {
+				expect(response.body).toEqual({
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "I find this existence challenging",
+        created_at: '2020-07-09T20:11:00.000Z',
+        votes: 100,
+        article_id: 1
+      })
+			})
+	})
+	it('Should be able to status 404 when a valid request is put through that is not available', () => {
+		return request(app)
+			.get('/api/articles/2222')
+			.expect(404)
+			.then( (response) => {
+				expect(response.body.msg).toBe('No user found')
```

Comment:
this is a good test - I might just reconsider the `msg` "No user found" because we're not looking for a user here they're looking for an article